docs/source/layers/index.rst

```diff
@@ -0,0 +1,32 @@
+Layers
+======
+
+.. toctree::
+    :maxdepth: 1
```

Comment:
```suggestion
    :maxdepth: 0
```

So we hopefully don't see the `Examples`, `Usage`, `Attributes` menus in the main page?